src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -258,14 +258,7 @@ EOF
 
   @Override
   def securityScan(){
-    if (steps.fileExists(".ci/security.sh")) {
```

Comment:
why remove this?

Reply:
Will add it back in later, just wanted to force it to use the shared script